Controllers/LoginController.cs

```diff
@@ -15,11 +15,16 @@ public IActionResult Index()
 
         public IActionResult ShowLogin(LoginModel login)
         {
+            //temp way to check authentication and redirect to user landing page
+            if (_accountService.AuthenticateUser(login))
+            {
+                //needs the .. to go back to local host/default controller
+                //
+                Response.Redirect("../userlanding");
+            }
             // TODO: Determine if UserModel is just better to use here.
-            return _accountService.AuthenticateUser(login) ?
-                View("LoginSuccess")
-                :
-                View("LoginFailure");            
+            return View("LoginFailure");
+
```

Comment:
This `Response.Redirect(...)` bleeds into the final return even on success. 

We should `return Response.Redirect(..)`.

A more explicity soultion would utilize `RedirectToAction(...)` to explicitly define where we are sending the User.

```c#
return RedirectToAction("Index", "UserLanding");
``` 
Optionally as a ternary statement
```c#
return _accountService.AuthenticateUser(login) ?
  RedirectToAction("Index", "UserLanding")
  :
  View("LoginFailure");
```